lib/wickedbase.pm

```diff
@@ -119,6 +119,7 @@ sub get_ip {
         'bond'         => ['10.0.2.18',            '10.0.2.17'],
         'dhcp'         => ['10.0.2.16',            '10.0.2.17'],
         'second_card'  => ['10.0.3.11',            '10.0.3.12'],
+        'dhcp_2nic'    => ['10.0.3.15',            '10.0.3.16'],
```

Comment:
`dhcp` entry was added exactly for this test case and not used anywhere else so instead introducing new entry here you need just modify `dhcp`

Reply:
Ok